package.json

```diff
@@ -24,17 +25,24 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "vite build"
+    "build": "tsc && vite build"
```

Comment:
Why do we have both tsc and vite for compiling? For declarations generation we can use `vite-plugin-dts` like in Image Tool
https://github.com/editor-js/image/blob/master/vite.config.js